src/qforte/adapters/molecule_adapters.py

```diff
@@ -12,23 +12,26 @@
 from qforte.system.molecular_info import Molecule
 from qforte.utils import transforms as tf
 
-from openfermion.ops import FermionOperator, QubitOperator
 
 try:
     from openfermion.chem import MolecularData
-except:
-    from openfermion.hamiltonians import MolecularData
-
-from openfermion.transforms import get_fermion_operator, jordan_wigner
-
-try:
+    from openfermion.transforms import get_fermion_operator, jordan_wigner
     from openfermion.transforms.opconversions import normal_ordered
     from openfermion.transforms.repconversions import freeze_orbitals
     from openfermion.utils import hermitian_conjugated
 except:
-    from openfermion.utils import hermitian_conjugated, normal_ordered, freeze_orbitals
+    try:
+        from openfermion.hamiltonians import MolecularData
+        from openfermion.transforms import get_fermion_operator, jordan_wigner
+        from openfermion.utils import hermitian_conjugated, normal_ordered, freeze_orbitals
+    except:
+        use_openfermion = False
```

Comment:
Why do we have this variable if we never use its value?

Reply:
New commit adds lines that raise import errors.